cdatransform/transform/gdclib.py

```diff
@@ -72,6 +72,10 @@ def diagnosis(tip, orig, log: LogValidation, **kwargs):
     for d in orig.get("diagnoses", []):
         this_d = {f: d.get(f) for f in diagnosis_fields}
         this_d["id"] = this_d.pop("diagnosis_id")
+        this_d["primary_diagnosis"] = this_d["primary_diagnosis"]
```

Comment:
I think these fields are already handled by the code on line 73:
```
        this_d = {f: d.get(f) for f in diagnosis_fields}
```

Reply:
You are correct. I just removed to lower function and didn't realize this line is completely useless